mem0/memory/main_graph.py

```diff
@@ -228,10 +228,10 @@ def get_all(self):
 
         # return all nodes and relationships
         query = """
-        MATCH (n)-[r]->(m)
+        MATCH (n)-[r]->(m) WHERE toLower(n.name) = toLower($name)
```

Comment:
Hi @rajib76, this is wrong. This will only return then nodes which are connected to the root node or the user_id node. For instance, the relation: alice123 loves pizza and john loves pizza. Your changes will just return alice123 loves pizza even if all of these nodes have been added using the same user_id.